### DISCUSSION TITLE
Updating inventory's quantity deletes local prices

### DISCUSSION THREAD

**Discussion by nmrdxt** [etsy/open-api#935] [Q&A]
Hello,

When running an inventory update, where only the quantity is updated, one of our merchants reports that the local prices are being removed from the variations.

The way the inventory update is performed is by:
1. Getting the current inventory from Etsy.
2. Goings through the products and only updating the quantity.
3. Sending the inventory array back to Etsy.

The result, as being reported, is that the local prices are removed from the products. Which is not good.

I do not have a concrete example to share, as I'm still attempting to replicate the issue, but the merchant has confirmed that it happened multiple times. 

Is there a known issue related to local prices being removed when inventory is updated?

Thanks!

**Comment by showtroylove** [comment]
I don't know about local prices, but prices in general appear to be deleted if updates are applied to that section of the listing.  You can observe this using their variations editor (see Exhibit A).  I'm working on this part of the API now.  I plan on doing a put of the entire object (no choice it's a PUT op).  You have to provide your JSON, at a minimum, for us to provide an assist.

**Exhibit A**
![image](https://user-images.githubusercontent.com/14541229/229943129-e6297ea6-8879-490f-b82b-8b390196bb9f.png)
